src/wilcoxon.jl

```diff
@@ -51,10 +51,10 @@ function signedrankstats{S<:Real}(x::AbstractVector{S})
    (W, ranks, nonzero_x .> 0, tieadj, length(x), median(x))
 end
 
-
 ## EXACT WILCOXON SIGNED RANK TEST
 
 immutable ExactSignedRankTest{T<:Real} <: HypothesisTest
+    vals::Vector            # original values
```

Comment:
one possible alternative to this is for me to just compute what I need for CI on initialization and just add it as a field here instead of retaining it as a field on the type.